package.json

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
I am not sure, but I think this file is not required. @branislaav please confirm it.

Reply:
I confirm it @dejwCake